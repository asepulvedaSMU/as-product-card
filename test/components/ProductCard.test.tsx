import * as React from 'react';
import renderer from 'react-test-renderer';
import { ProductCard } from '../../src/components';
import { product1 } from '../data/products';
describe('ProductCard', () => {
  test('debe mostrar el componente correctamente', () => {
    const wrapper = renderer.create(
      <ProductCard product={product1}>
        {() => <h1>Product card</h1>}
      </ProductCard>
    );

    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
