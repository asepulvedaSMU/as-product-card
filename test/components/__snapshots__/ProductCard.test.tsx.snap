// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductCard debe mostrar el componente con el nombre del producto 1`] = `
<div
  className="productCard undefined"
>
  <span
    className="productDescription undefined"
  >
    Coffee Mug - Sin imagen
  </span>
</div>
`;

exports[`ProductCard debe mostrar el componente correctamente 1`] = `
<span
  className="productDescription undefined"
>
  custom product
</span>
`;

exports[`ProductTitle debe mostrar el componente con el nombre del producto 1`] = `
<div
  className="productCard undefined"
>
  <span
    className="productDescription undefined"
  >
    Coffee Mug - Sin imagen
  </span>
</div>
`;

exports[`ProductTitle debe mostrar el componente correctamente con el titulo personalizado 1`] = `
<span
  className="productDescription undefined"
>
  custom product
</span>
`;
